.dockerignore

```diff
@@ -0,0 +1,17 @@
+node_modules
```

Comment:
Maybe this file can live under whatever appropriate sub-directory (`enclave`, `server`, `limbo`, ...) in future.

Reply:
Currently requires building the upstream arcs deps, so has to be here for now.  Agreed that this should live in server once arcs provides stable APIs.